test/unit/utils/fs.spec.ts

```diff
@@ -15,11 +15,13 @@
 import { homedir } from 'os';
 import { join } from 'path';
 
-import { extractTarBall, exists, rmdir, resolveAbsolutePath } from '../../../src/utils/fs';
+import { extractTarBall, exists, rmdir, resolveAbsolutePath, copyDir } from '../../../src/utils/fs';
 
 const testDir = `${process.cwd()}/test/data`;
 const tarFilePath = `${process.cwd()}/test/unit/fixtures/blockchain.db.tar.gz`;
 
+afterAll(async () => rmdir(testDir, { force: true, recursive: true }));
```

Comment:
would it be better to clean the directory on `beforeAll` as well to handle the case when a previous test crashed and left undeleted data?